packages/lib/src/utils/createDataProp.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
I think this is safe to rm .. it's not used anywhere else in the leafy codebase. What do you think @DesignerDave

Reply:
Removing this would technically make this a major change instead of just a patch. Not saying we can't, but that's a consideration

Reply:
We gotta remove it at some point 🤷‍♂️